config/locales/en.yml

```diff
@@ -72,20 +72,52 @@ en:
       custom_select_error: Select the best way to describe your business
     product_details:
       title: Tell us about each product you're offering
```

Comment:
```suggestion
      title: Tell us about the product you’re offering
```

Reply:
I'm going to do this in another PR, since there are other updates to this bit: https://trello.com/c/koG4jHHw/113-change-format-and-flow-of-question-1

Reply:
👍  I was just looking at that card